Sources/Sentry/include/SentryDefines.h

```diff
@@ -6,6 +6,8 @@
 //  Copyright © 2017 Sentry. All rights reserved.
 //
 
+#import <Foundation/Foundation.h>
```

Comment:
This was an implicit dependency (`NSError` is used below) but I've now made it explicit as it could be the first import in a file.